src/main/kotlin/thatline/localup/chatgpt/service/ChatGptService.kt

```
@@ -41,26 +52,200 @@ class ChatGptService(
+                ${currentWeather?.let {
+                    """- 현재 상황: ${it.time}시 기준
+                - 기온: ${it.temperature?.toInt()}°C
+                - 습도: ${it.humidity}%
+                - 강수확률: ${it.precipitationProbability}%
+                - 하늘상태: ${it.skyCondition?.label ?: "정보없음"}"""
+                } ?: ""}
+                """.trimIndent()
+            } else {
+                "\n[현재 날씨 정보를 불러올 수 없습니다]"
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)